src/Aspire.ProjectTemplates/templates/aspire-mstest/.template.config/template.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "https://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    ".NET Aspire",
+    "Cloud",
+    "Web",
+    "Web API",
+    "API",
+    "Service",
+    "Test"
+  ],
+  "name": ".NET Aspire Test Project (MSTest)",
+  "defaultName": "Tests",
+  "description": "A project that contains MSTest integration tests of a .NET Aspire AppHost project.",
+  "shortName": "aspire-mstest",
+  "sourceName": "Aspire.Tests1",
+  "preferNameDirectory": true,
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "precedence": "8000",
+  "identity": "Aspire.Tests.MSTest.CSharp.8.0",
```

Comment:
It's not clear to me why with a different identity I have some conflict when trying to run the template:

![image](https://github.com/dotnet/aspire/assets/11340282/41c2fbaa-9f6c-4cad-8763-7f1a36dc2717)

Reply:
Solved thanks to @baronfel (group identity needs to differ too).